hooks/useMetaplex.ts

```diff
@@ -0,0 +1,19 @@
+import { Metaplex, walletAdapterIdentity } from '@metaplex-foundation/js'
+import { useWallet } from '@solana/wallet-adapter-react'
+import mangoStore from '@store/mangoStore'
+import metaplexStore from '@store/metaplexStore'
+import { useEffect } from 'react'
+
+export default function useMetaplex() {
+  const connection = mangoStore((s) => s.connection)
+  const wallet = useWallet()
```

Comment:
same

Reply:
fixed other places but in here im using whole wallet for metaplex identity